cdap-app-fabric/src/main/java/co/cask/cdap/metadata/DefaultMetadataAdmin.java

```diff
@@ -45,6 +45,13 @@
     .or(CharMatcher.is('_')
           .or(CharMatcher.is('-')));
 
+  private static final CharMatcher valueMatcher = CharMatcher.inRange('A', 'Z')
```

Comment:
static final field name should be ALL_CAPS, right? Looks like checkstyle flags only if field is public.